util/chplenv/chpl_llvm.py

```diff
@@ -704,8 +704,7 @@ def get_gcc_prefix():
 def get_sysroot_resource_dir_args():
     args = [ ]
     target_platform = chpl_platform.get('target')
-    llvm_val = get()
-    if (target_platform == "darwin" and llvm_val == "bundled"):
+    if target_platform == "darwin":
```

Comment:
If the homebrew LLVM 18 package can't compile C / C++ code due to this then it's arguably an error in the homebrew package.

Reply:
Yes I do prefer it to be restored, thanks. You might consider giving this behavior an environment variable, though, that you (and others) can use in such situations.